code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -116,7 +116,19 @@
 	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 10, bio = 100, rad = 0)
 	max_heat_protection_temperature = FIRE_IMMUNITY_HELM_MAX_TEMP_PROTECT
 	flash_protect = 2
-	
+
+/obj/item/clothing/suit/space/eva/plasmaman/mechanic
+	name = "plasmaman mechanic suit"
```

Comment:
Wait, aren't these guys already space worthy? Unless this is their actual uniform which didn't exist until now